packages/cactus-cmd-api-server/tsconfig.json

```diff
@@ -11,7 +11,7 @@
   "include": [
     "./src",
     "src/**/*.json"
-  ],
+, "src/main/typescript/openapi/.ts"  ],
```

Comment:
@charellesandig Why was this addition necessary? Don't the existing include patterns cover this path already?